src/config.json

```diff
@@ -1,5 +1,8 @@
 {
 	"port": 8080,
 	"bodyLimit": "100kb",
-	"corsHeaders": ["Link"]
+	"corsHeaders": ["Link"],
+    "mysql_user": "root",
```

Comment:
add this file to `.gitignore` and then add an `example.config.json` to show what it should look like without the private information